command/watch.go

```diff
@@ -182,11 +185,19 @@ func (c *WatchCommand) Run(args []string) int {
 			cmd.Stdout = os.Stdout
 			cmd.Stderr = os.Stderr
 
+			// Watch for a stop
+			go func() {
+				<-wpStoppingCh
+				cmd.Process.Kill()
+				wpStoppedCh <- cmd.Wait()
```

Comment:
I'd probably just handle the error here by logging the message (if any). It would also be more helpful to check the error from the call to `Kill()`. We could then change the `wpStoppedCh` to a normal `chan struct{}` and just close it for signaling completion. Alternatively, we could use a `sync.WaitGroup`, but that's not entirely necessary.